doc/supportedPlatformVersions.md

```diff
@@ -912,6 +915,7 @@
 - 8.1.4
 - 8.1.6
 - 8.1.9
+- 8.2.0
```

Comment:
Just curious about this one: is a consequence of adding this to the 'stretch' versionToBuild, so that even older versions of App Service can still get the new update?

Reply:
Yep, since the stretch version tarballs are not in the debian-debian-flavor format, the App Service partners should be able to use this SDK with their older versions as well.
Additionally, the validation tests will fail if the new version is not added to the stretch versionToBuild.txt file since the final stretch version for php changes have not been released yet.

Reply:
OK thanks